assets/src/styles/blocks/core-overrides/BlockSpacing.scss

```diff
@@ -23,6 +21,21 @@
   }
 }
 
+.page-content > .wp-block-group,
+.page-content > .wp-block-media-text {
+  margin-top: $sp-1;
+  margin-bottom: $sp-2;
+
+  @include large-and-up {
+    margin-top: $sp-2;
+    margin-bottom: $sp-4;
+  }
+
+  &:first-child {
```

Comment:
Do we still need this, even with the new PR? If not then we probably need no changes in this file 🤔

Reply:
I think we should keep this change. Here is [another demopage ](https://www-dev.greenpeace.org/test-saturn/1429-2/)(which is similar of the reported). See what happen if you remove the margin top. 
What do you think?